showcase/src/app/demo-rbac/services/demo-authorization.service.ts

```diff
@@ -2,7 +2,7 @@ import { StarkRBACAuthorizationServiceImpl } from "@nationalbankbelgium/stark-rb
 import { StarkUser } from "@nationalbankbelgium/stark-core";
 import { Injectable } from "@angular/core";
 
-@Injectable()
+@Injectable({ providedIn: "root" })
```

Comment:
Weird that this service was not included in the  `providers` of the module and yet it was working!! 😲

Reply:
You're right ^^
Sometimes, angular is magic :sparkles:

Reply:
Magic or evil ? :-)